src/pages/City/AddDocumentModal/AddDocumentModal.tsx

```diff
@@ -177,7 +181,7 @@ const AddDocumentModal = (props: Props) => {
           </Form.Item>
 
           <Form.Item name="datepicker" label="Дата документу">
-            <DatePicker format="DD.MM.YYYY" className="formSelect" />
+            <DatePicker disabledDate={disabledDate} format="DD.MM.YYYY" className="formSelect" />
```

Comment:
It is better to move disabled date in some global folder, and then use it everywhere.